x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/expression/UnresolvedAttribute.java

```diff
@@ -107,6 +107,11 @@ protected String label() {
         return UNRESOLVED_PREFIX;
     }
 
+    @Override
+    public boolean isDimension() {
+        return false;
```

Comment:
thought: maybe this should throw UOE, because we can't determine this on an unresolved attr.